tests/unit/neptune/new/attributes/series/test_float_series.py

```diff
@@ -51,3 +52,7 @@ def test_log(self):
             values = list(var.fetch_values()["value"].array)
             expected = list(range(0, 5000))
             self.assertEqual(len(set(expected)), len(set(values)))
+
+    def test_handle_stringified_value(self):
```

Comment:
This looks a little bit weird as we're mocking some internal value. Isin't it already covered in stringify value related test?